### PR TITLE
CB-67-Add-slider

### PR DESCRIPTION
**Pull request by banaskrzysztof**

### CODE REVIEW

**Review by KrzysiekLipiec — APPROVED** [review]

File: src/app/modules/recipe-list/components/recipe-carousel.component.ts [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+@Component({
+  selector: 'app-recipe-carousel',
+  templateUrl: './recipe-carousel.component.html',
+  styleUrls: ['./recipe-carousel.component.scss']
+})
+export class RecipeCarouselComponent {
+
+  public limit = 3;
+  images = [
```
Comment (tomekknowde) [inline src/app/modules/recipe-list/components/recipe-carousel.component.ts:11]: use always public
```suggestion
  public limit = 3;
  public images = [
```
or never do that (make your code consistent)
```suggestion
  limit = 3;
  images = [
```

File: src/app/modules/recipe-list/components/recipe-carousel.component.ts [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    {path: 'https://ocdn.eu/pulscms-transforms/1/Et5k9kpTURBXy8zYTg1NGQ5YTUwNmQ3MGUzNWVmM2E2N2FmOGJiNWM2Yy5qcGeTlQMAzKLNFEDNC2STBc0DFM0BvJMJpjZiZjgyYgaBoTAB/pierogi-z-kapusta.jpg'}
+  ]
+
+  @HostListener('window:resize', ['$event'])
+  onResize(event: { target: { innerWidth: number; }; }) {
+    if (event.target.innerWidth < 768) {
+      this.limit = 1;
+    } else {
+      this.limit = 3;
+    }
```
Comment (tomekknowde) [inline src/app/modules/recipe-list/components/recipe-carousel.component.ts:27]: Create enum for screen breakpoints:

```
export enum BreakpointsEnum {
  xs = 576,
  sm = 768,
  md = 992,
  lg = 1200
}
```

```suggestion
    this.limit = event.target.innerWidth < BreakpointsEnum.sm ? 1 : 3;
```

File: src/app/modules/recipe-list/components/recipe-carousel.component.ts (hunk 1 above)
Comment (tomekknowde) [inline src/app/modules/recipe-list/components/recipe-carousel.component.ts:11]: images should goes from parent component, change to 
```suggestion
  @Input() images: Array<PleaseCreateProperTypeHere> = [];
```

File: src/app/modules/recipe-list/components/recipe-carousel.component.ts [hunk 3]
```diff
@@ -0,0 +1,29 @@
+import {Component, HostListener, OnInit} from '@angular/core';
+
+@Component({
+  selector: 'app-recipe-carousel',
+  templateUrl: './recipe-carousel.component.html',
+  styleUrls: ['./recipe-carousel.component.scss']
+})
+export class RecipeCarouselComponent {
```
Comment (tomekknowde) [inline src/app/modules/recipe-list/components/recipe-carousel.component.ts:8]: To be honest, this component don't make big sense, we can just use `<carousel>` inside `recipe-list-component`

**Review by tomekknowde — CHANGES_REQUESTED** [review]

**Review by tomekknowde — APPROVED** [review]

**Pull request merged**